server/management/sites/sites.php

```diff
@@ -82,7 +82,9 @@ function getAppointmentTimesForSite($siteId, $year = null) {
 		FROM AppointmentTime
 		JOIN AppointmentType ON AppointmentTime.appointmentTypeId = AppointmentType.appointmentTypeId
 		LEFT JOIN Appointment ON AppointmentTime.appointmentTimeId = Appointment.appointmentTimeId
+        LEFT JOIN ServicedAppointment ON Appointment.appointmentId = ServicedAppointment.appointmentId
 		WHERE siteId = ?
+			AND (cancelled IS NULL OR cancelled = FALSE)
```

Comment:
Ohhh I see, the appointment times weren't _actually_ overbooked, they just appeared to be cuz the number e were reporting on the "Sites Management" page was wrong?

Reply:
Yep